internal/nvidianetworkconfig/config.go

```diff
@@ -0,0 +1,35 @@
+package nvidianetworkconfig
+
+import (
+	"log"
+
+	"github.com/kelseyhightower/envconfig"
+)
+
+// NvidiaNetworkConfig contains environment information related to nvidianetwork tests.
+type NvidiaNetworkConfig struct {
+	CatalogSource                      string `envconfig:"NVIDIANETWORK_CATALOGSOURCE"`
+	SubscriptionChannel                string `envconfig:"NVIDIANETWORK_SUBSCRIPTION_CHANNEL"`
+	CleanupAfterTest                   bool   `envconfig:"NVIDIANETWORK_CLEANUP" default:"true"`
+	DeployFromBundle                   bool   `envconfig:"NVIDIANETWORK_DEPLOY_FROM_BUNDLE" default:"false"`
+	BundleImage                        string `envconfig:"NVIDIANETWORK_BUNDLE_IMAGE"`
+	OperatorUpgradeToChannel           string `envconfig:"NVIDIANETWORK_SUBSCRIPTION_UPGRADE_TO_CHANNEL"`
+	NNOFallbackCatalogsourceIndexImage string `envconfig:"NVIDIANETWORK_NNO_FALLBACK_CATALOGSOURCE_INDEX_IMAGE"`
+	NFDFallbackCatalogsourceIndexImage string `envconfig:"NVIDIANETWORK_NFD_FALLBACK_CATALOGSOURCE_INDEX_IMAGE"`
+}
+
+// NewNvidiaNetworkConfig returns instance of NvidiaNetworkConfig type.
+func NewNvidiaNetworkConfig() *NvidiaNetworkConfig {
+	log.Print("Creating new NvidiaGPUConfig")
```

Comment:
I wonder why not use "glog" here too unless you want to print it always

Reply:
This was also in the internal/nvidiagpuconfig/config.go, did not want to change, may be next PR will update both ?